test/test_linalg.py

```diff
@@ -125,12 +125,12 @@ def _hip_allow_tf32(self):
                 del os.environ["HIPBLASLT_ALLOW_TF32"]
 
     def setUp(self):
-        super(self.__class__, self).setUp()
+        super().setUp()
```

Comment:
Wow, that's a remnants of py2->py3-migration 
To be a bit pedantic, do you want to land it as BE change in separate PR?

Reply:
> To be a bit pedantic, do you want to land it as BE change in separate PR?

I would split it but it's required to get CI passing sadly :/